src/database/DataTypes/Location.js

```diff
@@ -24,7 +24,7 @@ export class Location extends Realm.Object {
   }
 
   get currentTemperature() {
-    const { temperature } = this.mostRecentTemperatureLog;
+    const { temperature } = this.mostRecentTemperatureLog ?? {};
```

Comment:
Just checking intended behaviour is to return `null` instead of a default?

Reply:
yeah - default I think is a bit nasty - can't be `0`, `-Infinity` or `Infinity` are just more annoying to check, especially now with `??`